cluster-autoscaler/provisioningrequest/apis/go.mod

```diff
@@ -0,0 +1,47 @@
+module sigs.k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/apis
```

Comment:
I think the module should be
`sigs.k8s.io/autoscaler/cluster-autoscaler/apis`

where one of the folders is `provisioningrequest`.

Reply:
The reason being that we might have an generic (independent of the cloud provider) implementation of `provisioningrequest` and it might have dependencies that are not worth leaking to users of the API.

Reply:
I see. Maybe, we can put a client folder on `sigs.k8s.io/autoscaler/cluster-autoscaler/apis/provisioningrequest/client` as well.

Reply:
Doesn't `sigs.k8s.io` resolve to kubernetes-sigs org? This repo is in main kubernetes org, so shouldn't it be under `k8s.io/autoscaler/cluster-autoscaler`?

Reply:
Oops. Sorry, you're right.